src/internal/m365/collection/drive/collections_test.go

```diff
@@ -1175,11 +1175,13 @@ func (suite *CollectionsUnitSuite) TestPopulateDriveCollections() {
 				errs     = fault.New(true)
 			)
 
-			mbh.DriveItemEnumeration = mock.EnumerateItemsDeltaByDrive{
-				DrivePagers: map[string]*mock.DriveItemsDeltaPager{
+			mbh.DriveItemEnumeration = mock.EnumerateDriveItemsDelta{
```

Comment:
lots of chaff here.  all changes in this file should be the following refactor:
* `mock.EnumerateItemsDeltaByDrive` -> `mock.EnumerateDriveItemsDelta`
* `DrivePagers: map[string]*mock.DriveItemsDeltaPager` -> `DrivePagers: map[string]*mock.DeltaIterator`
* addition of `}},` at the block end